Formula/ksync.rb

```diff
@@ -28,7 +28,7 @@ def install
       -w
       -X #{project}/pkg/ksync.GitCommit=#{Utils.git_short_head}
       -X #{project}/pkg/ksync.GitTag=#{version}
-      -X #{project}/pkg/ksync.BuildDate=#{Utils.safe_popen_read("date", "+%Y%m%dT%H%M%S").chomp}
+      -X #{project}/pkg/ksync.BuildDate=#{DateTime.now.utc.rfc3339(9)}
```

Comment:
```suggestion
      -X #{project}/pkg/ksync.BuildDate=#{Time.now.utc.rfc3339(9)}
```

No strong preference but we may as well be consistent

Reply:
Didn't realise ActiveSupport supplied this. :+1:

Reply:
Cherry-picked the other commits, left this change here for testing.